schemas/src/main/java/com/scottlogic/deg/schemas/v0_1/NoopProfileSchemaValidator.java

```diff
@@ -0,0 +1,27 @@
+package com.scottlogic.deg.schemas.v0_1;
+
+import com.scottlogic.deg.schemas.common.ValidationResult;
+
+import java.io.File;
+import java.io.InputStream;
+import java.util.Collections;
+
+public class NoopProfileSchemaValidator implements ProfileSchemaValidator {
```

Comment:
There is a Cucumber schema validator which does exactly the same as this, can the cucumber one be renamed, or it be removed and use this type instead.